lib/mongoid/association/depending.rb

```diff
@@ -10,7 +10,7 @@ module Depending
 
       included do
         def self.dependents
-          @dependents ||= []
+          @dependents ||= superclass.method_defined?(:dependents) ? superclass.dependents.dup : []
```

Comment:
Wouldn't this then modify superclass' dependents if both super and subclass define dependents?

Reply:
I don't think so? The actual method call is `superclass.dependents.dup`, so it isn't reusing the reference to the superclass's array.

Reply:
Good point! Let me know what you think of the changes I just pushed